trick_source/data_products/Log/MatLab.cpp

```diff
@@ -111,7 +111,7 @@ MatLab::MatLab(char * file_name, char * param_name, char * time_name) {
             swap_ = 0;
         } else if (magic == 0x494d) {
             swap_ = 1;
-        }
+        } 
```

Comment:
Suggest to undo the added trailing whitespace

```suggestion
        }
```

Reply:
fixed, thanks